### PR TITLE
Add storytellers to before you ship

### PR DESCRIPTION
**Pull request by melodykramer**
Adds guidance for product team on storytellers initiative which will act as a conduit between product and comms for all projects.

### CODE REVIEW

**Comment by afeld** [comment]
@leahbannon @annahsebok Will defer to y'all.

File: _pages/index.md [hunk 1]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
 
 ### Outreach
 
+* Appoint a storyteller for your project early in the cycle and notify #outreach. The storyteller will be your project's conduit with the outreach team and should notify the outreach team about:
```
Comment (leahbannon) [inline _pages/index.md]: Suggest shortening 

> The storyteller will be your project's conduit with the outreach team

to

The storyteller will coordinate with the outreach team

File: _pages/index.md [hunk 2]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
 
 ### Outreach
 
+* Appoint a storyteller for your project early in the cycle and notify #outreach. The storyteller will be your project's conduit with the outreach team and should notify the outreach team about: 
+
+    - upcoming milestones and events (that you’re going to or could present at)
+    - any impending risks to prepare for press and mitigate any issues
```
Comment (leahbannon) [inline _pages/index.md]: remove "any"s?

File: _pages/index.md [hunk 3]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
 
 ### Outreach
 
+* Appoint a storyteller for your project early in the cycle and notify #outreach. The storyteller will be your project's conduit with the outreach team and should notify the outreach team about: 
+
+    - upcoming milestones and events (that you’re going to or could present at)
```
Comment (leahbannon) [inline _pages/index.md]: perhaps edit 

> (that you’re going to or could present at)

to

"(attending or presenting)

File: _pages/index.md [hunk 4]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
...
+    - any impending risks to prepare for press and mitigate any issues
+    - progress on comms plan / blog posts
```
Comment (leahbannon) [inline _pages/index.md]: The communications plan is discussed below. Can we consolidate?

File: _pages/index.md [hunk 5]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
...
+    - any other ways the comms team can help your project
+    - upcoming speaking events / conferences that the project/product is represented at
```
Comment (leahbannon) [inline _pages/index.md]: this seems redundant to "upcoming milestones and events" -- maybe just add "/conferences" to that?

File: _pages/index.md [hunk 6]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
...
+    - any updates that need to be conveyed to the agency partner’s comms team (and who those people might be.)
```
Comment (leahbannon) [inline _pages/index.md]: remove "any"?

File: _pages/index.md [hunk 7]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
...
+    - updates for 18F newsletter or Dolores or #news
```
Comment (leahbannon) [inline _pages/index.md]: suggest edit to "newsletter, the Dolores bot, or #news"

File: _pages/index.md [hunk 8]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
...
+    - updates for open source community who might be able to help with ‘help-wanted’ tags
```
Comment (leahbannon) [inline _pages/index.md]: I don't understand how this should work. Do you mean that we should use "help wanted" tags and then notify the comms team, who will help publicize them in the open source community?

File: _pages/index.md [hunk 9]
```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
...
 * Are your project's details up to date in the [18F Team API](https://github.com/18F/team-api.18f.gov)? Make that happen by [creating an .about.yml file](https://github.com/18F/team-api.18f.gov#adding-project-data); the outreach team recommends creating this at the beginning of your project and updating it monthly. Good data in the Team API supports future iterations of the [Dashboard](https://18f.gsa.gov/dashboard/) and other useful efforts.
-* You need to share details about your project and develop a communications plan with our outreach team (who will also help coordinate with the necessary GSA front offices, such as press and congressional affairs). To do this, complete this [communications plan template](https://docs.google.com/document/d/1xc7H6m7lfesCN-phJGvGSDPmtoinB5sM9KAA8deMNTQ/edit).
+* Your storyteller will need to share details about your project and develop a communications plan with our outreach team (who will also help coordinate with the necessary GSA front offices, such as press and congressional affairs). To do this, complete this [communications plan template](https://docs.google.com/document/d/1xc7H6m7lfesCN-phJGvGSDPmtoinB5sM9KAA8deMNTQ/edit).
```
Comment (leahbannon) [inline _pages/index.md]: Perhaps we could merge this section with the above section? Maybe something like: 

Appoint a storyteller for your project early and then notify #outreach. The storyteller will coordinate with the outreach team and work with them to develop a [communications plan template](https://docs.google.com/document/d/1xc7H6m7lfesCN-phJGvGSDPmtoinB5sM9KAA8deMNTQ/edit). This is required before launches, when the outreach team will also help coordinate with the necessary GSA front offices, such as press and congressional affairs. Your storyteller will also notify the outreach team about:

File: _pages/index.md (hunk 6 above)
Comment (melodykramer) [inline _pages/index.md]: the latter. The idea is to tell us if we need to coordinate with an external comms team, which we often do for releases.

File: _pages/index.md (hunk 8 above)
Comment (melodykramer) [inline _pages/index.md]: We add tasks that are requesting external support to our biweekly external newsletter to give them an additional boost. We'd like teams to let us know if there's an issue that we can surface to these external audiences.

**Comment by melodykramer** [comment]
edits have been added @leahbannon

**Comment by annahsebok** [comment]
Thanks for the FYI @afeld. This looks awesome but it's probably also worth mentioning that this is already documented for product leads [in the Product Guide](https://pages.18f.gov/product-guide/lifecycle-of-a-project/project-communications/) thanks to the wonderful @abisker. I defer to @leahbannon & @afeld as to whether or not this also belongs in the *Before You Ship* guide as well since I didn't take part in writing *Before You Ship*.

**Comment by afeld** [comment]
> this is already documented for product leads in the Product Guide

Hmmmmm, very good call. See https://github.com/18F/before-you-ship/pull/151.

**Comment by afeld** [comment]
Closing in favor of #151.

**Pull request closed without merging**